pkg/charts/linkerd2/values.go

```diff
@@ -79,6 +81,17 @@ type (
 		ProxyInjectorProxyResources *Resources `json:"proxyInjectorProxyResources"`
 	}
 
+	// DeploymentStrategy contains the Kubernetes strategy configuration
+	DeploymentStrategy struct {
```

Comment:
Is there a DeploymentStrategy type from apimachinery that we can reuse here?

Reply:
Pulled from k8s/apis